Bottom: 9ae4117ae645aaa227c437059236b537aa21e207
Top:    d7ae62f57194a054e2c7ec9491ffab7041cdf38d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-13 14:31:55 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index eb4cac2..73468d2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2748,7 +2748,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
